.ci/Jenkinsfile

```diff
@@ -362,7 +321,8 @@ def runScript(Map params = [:]){
   def framework = params.framework
   log(level: 'INFO', text: "${label}")
   retry(2){
-    withEnv(["HOME=${env.WORKSPACE}", "PATH=${env.PATH}:${env.WORKSPACE}/bin"]) {
+    withEnv(["HOME=${env.WORKSPACE}", "PATH=${env.PATH}:${env.WORKSPACE}/bin",
+             "RUBY_VERSION=${ruby}"]) {
```

Comment:
Just in case the `RUBY_VERSION` env variable might be needed, though maybe it should not be required..